Dataflow

danagilliann opened the pull request:
Adding the dataflow flow part to app.

bettyybei committed:
starting with scalica

ashleytqy committed:
set up controller, routes and views, without any logic atm

ashleytqy committed:
using bootstrap, overriding default scalica copy

ashleytqy committed:
fake content

ashleytqy committed:
add two missing templates

ashleytqy committed:
link to user page

ashleytqy committed:
remove custom css

ashleytqy committed:
empty post controllers

ashleytqy committed:
follow and unfollow buttons

laurachevalier4 committed:
adding back in models.py, commenting out Yair's Scalica models/forms

laurachevalier4 committed:
adding forms for user, profile, answer, and question

laurachevalier4 committed:
add form for school

laurachevalier4 committed:
replace 'school' with 'university'

laurachevalier4 committed:
fixed spacing issues(?)

laurachevalier4 committed:
adding necessary imports

bettyybei committed:
fix model and url compile errors

bettyybei committed:
posting question works

bettyybei committed:
posting questions and answers work

bettyybei committed:
cleanup

bettyybei committed:
cleanup

bettyybei committed:
cleanup

danagilliann committed:
first commit

danagilliann committed:
Starting out the pipeline

laurachevalier4 committed:
added default num_upvotes value and added user ref to Answer model

bettyybei committed:
fix answer

ashleytqy committed:
can follow but cant unfollow yet

danagilliann committed:
correctly running the pipeline

ashleytqy committed:
sytling

ashleytqy committed:
styling

bettyybei committed:
user can add profile

bettyybei committed:
user can add profile

ashleytqy committed:
styling

bettyybei committed:
university validation for question and answer forms

bettyybei committed:
cleanup

bettyybei committed:
cleanup

bettyybei committed:
add user names to question and answers

bettyybei committed:
cleanup

danagilliann committed:
triggering pipeline with python

laurachevalier4 committed:
update settings with cloud sql info (might be wrong...)

ashleytqy committed:
stuff

ashleytqy committed:
Revert "cleanup"

ashleytqy committed:
remove settings

ashleytqy committed:
Revert "Revert "cleanup""

ashleytqy committed:
add startup script

ashleytqy committed:
add script

ashleytqy committed:
s

ashleytqy committed:
x

ashleytqy committed:
dashboard

ashleytqy committed:
oops

ashleytqy committed:
add instructions to restart db

ashleytqy committed:
revert if any probs

bettyybei committed:
fix sorting and allow spaces in profile

bettyybei committed:
fix restart db instructions

laurachevalier4 committed:
adding icon

ashleytqy committed:
add instruction

ashleytqy committed:
add

ashleytqy committed:
emove sutf

laurachevalier4 committed:
accounting for duplicates in view/template

laurachevalier4 committed:
add text for dup

ashleytqy committed:
style

ashleytqy committed:
db setting changes

ashleytqy committed:
change startup script

ashleytqy committed:
include dataflow

remove get-pip

laurachevalier4 committed:
32 x 32 icon

ashleytqy reviewed (commented):
can you remove the files that are not part of `app/pipeline/dataflow`? also, what does `app/pipeline/dataflow/get-pip.py` do?

danagilliann commented:
@ashleytqy i squashed the commits and I think i cleaned up everything. i can't remove the non-pipeline files because i believe it will delete the entire file upon merging. I believe `app/pipeline/dataflow/get-pip.py` helps you manage pip but it runs fine without it so I deleted it
